email/debian/control

```diff
@@ -3,14 +3,14 @@ Section: mail
 Priority: extra
 Maintainer: James Carter <jcarter@bytemark.co.uk>
 Uploaders: Patrick J Cherry <patrick@bytemark.co.uk>, Steve Kemp <steve@bytemark.co.uk>
-Build-Depends: debhelper (>= 7.0.0), txt2man, gem2deb, dh-systemd
+Build-Depends: debhelper (>= 7.0.0), txt2man, gem2deb, dh-systemd, symbiosis-common
```

Comment:
The [hack of doom](https://github.com/BytemarkHosting/symbiosis/blob/jessie/common/lib/symbiosis/utils.rb#L37)